docs/docsite/rst/user_guide/become.rst

```diff
@@ -749,6 +749,13 @@ Limitations of become on Windows
 
 * The Secondary Logon service ``seclogon`` must be running to use ``ansible_become_method: runas``
 
+Resolving Temporary File Error Messsages
+----------------------------------------
+
+"Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user"
+* In Debian systems, this error can be resolved by installing the `acl` package, which provides the `setfacl` command.
```

Comment:
This seems too specific as it is not just Debian systems, is it?

Reply:
A more generic 'This error can be resolved by installing the system package that provides the `setfacl` command.' would be better and avoid the need to list every OS family/package name

Reply:
What about a second page to link to that would call out specific packages?

'This error can be resolved by installing the system package that provides the setfacl command. A list of compatible packages / resolutions can be found [here](https://mypretendlink.com)'

The overall goal would be, if I am someone who gets this error, I can google the error, and find a very clear trail to get to the answer, so I can follow whatever it says and keep working. Or, as I like to call it, the Stack Overflow School of Error Resolution. I can say from experience that currently figuring out the solution is a bit of a rabbit-hole, and I'd love to save someone else the same hassle.

Reply:
I would absolutely understand if the Ansible team put Fedora over Debian in terms of server distributions though. I'm just saying it makes sense to cover the vast majority of the user base.

Reply:
works4me